sound/soc/sof/Kconfig

```diff
@@ -198,13 +198,24 @@ config SND_SOC_SOF_DEBUG_ENABLE_FIRMWARE_TRACE
 	  If unsure, select "N".
 
 config SND_SOC_SOF_DEBUG_IPC_FLOOD_TEST
-	bool "SOF enable IPC flood test"
+	tristate "SOF enable IPC flood test"
+	select SND_SOC_SOF_CLIENT
 	help
-	  This option enables the IPC flood test which can be used to flood
-	  the DSP with test IPCs and gather stats about response times.
+	  This option enables a separate client device for IPC flood test
+	  which can be used to flood the DSP with test IPCs and gather stats
+	  about response times.
 	  Say Y if you want to enable IPC flood test.
 	  If unsure, select "N".
 
+if SND_SOC_SOF_DEBUG_IPC_FLOOD_TEST
```

Comment:
a depends on SND_SOC_SOF_DEBUG_IPC_FLOOD_TEST is enough, we use 'if' when there are more than one cases.

Reply:
Sure, switched to depends on.